frontend/src/page/ReceivedInvoice.jsx

```
@@ -313,39 +563,56 @@ function ReceivedInvoice() {
-                        <td className="border p-2">{ethers.formatUnits(item.unitPrice)}</td>
-                        <td className="border p-2">{item.discount.toString()}</td>
-                        <td className="border p-2">{item.tax.toString()}</td>
-                        <td className="border p-2">
-                          {ethers.formatUnits(item.amount)} ETH
-                        </td>
-                      </tr>
-                    </tbody>
-                  ))
-                }
+                {drawerState.selectedInvoice?.items?.map((item, index) => (
+                  <tbody>
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)